pubsub.go

```diff
@@ -13,11 +14,12 @@ import (
 
 	logging "github.com/ipfs/go-log"
 	crypto "github.com/libp2p/go-libp2p-crypto"
+	discovery "github.com/libp2p/go-libp2p-discovery"
 	host "github.com/libp2p/go-libp2p-host"
 	inet "github.com/libp2p/go-libp2p-net"
 	peer "github.com/libp2p/go-libp2p-peer"
 	protocol "github.com/libp2p/go-libp2p-protocol"
-	timecache "github.com/whyrusleeping/timecache"
+	"github.com/whyrusleeping/timecache"
```

Comment:
please don't remove this named import.

Reply:
umm... sure. Although I was actually planning on doing the opposite. From what I can tell since the go-libp2p-core refactor all of the named imports have been removed in favor of unnamed imports (if the names would match and not cause problems). e.g. look at go-libp2p-swarm

Reply:
for now we will use a policy of named imports for non-libp2p and non-standard packages, so `timecache` will go back in